docs/plugin-proposal-decorators.md

```diff
@@ -80,17 +80,31 @@ require("@babel/core").transformSync("code", {
 
 ## Options
 
-### `decoratorsBeforeExport`
-
-`boolean`
-
 <details>
   <summary>History</summary>
 | Version | Changes |
 | --- | --- |
-| `v7.2.0` | `decoratorsBeforeExport` must be specified. Before that it defaults to `false` |
```

Comment:
IMO the old history should be preserved. And we should also add that `legacy: true` has been deprecated since 7.17.

Reply:
It's actually required since 7.0.0 (https://github.com/babel/babel/commit/d79b5eeeffab14f89c5be92888965e57fcab6436). I was really surprised when I saw this history reporting a breaking change in a minor version.